website/static/js/commentpane.js

```diff
@@ -52,8 +61,16 @@
             } else {
                 var bodyWidth = $(document.body).width();
                 width = options.toggleWidth * bodyWidth;
-                options.onOpen.call(self);
+                var open = options.onOpen.call(self, self.page, self.hostName);
+                open.fail(function(xhr, textStatus, errorThrown) {
```

Comment:
`CommentPane` should not need to know that `onOpen` returns an jqXHR. This error handling logic can go back in `comment.js`.